release-note-script/src/main/java/ReleaseNoteCreation.java

```diff
@@ -0,0 +1,321 @@
+import static java.lang.String.format;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * ReleaseNoteCreation creates the body of the release note for a
+ * repository of Scalar products. The body of the release note is written
+ * out as a markdown to standard output.
+ * 
+ * This script is assumed to be executed in a GitHub Actions workflow.
+ * 
+ * Note that it is needed that java 11 to execute this script since it is
+ * executed as a Single-File Source-code program.
+ */
+public class ReleaseNoteCreation {
+
+    private final String DEBUG = System.getenv("DEBUG");
+
+    private GitHubContext ghContext;
+
+    Map<Category, List<ReleaseNoteText>> categoryMap = new HashMap<>();
```

Comment:
We can use `EnumMap` here?

Reply:
I use `EnumMap` instead of `HashMap` as much as possible. Thank you.